Cargo.toml

```diff
@@ -4,6 +4,6 @@ members = ["bin/mev", "mev-boost-rs", "mev-relay-rs", "mev-build-rs", "mev-rs"]
 default-members = ["bin/mev"]
 
 [workspace.dependencies]
-ethereum-consensus = { git = "https://github.com/ralexstokes/ethereum-consensus", rev = "2bcb975" }
-beacon-api-client = { git = "https://github.com/ralexstokes/beacon-api-client", rev = "d838d93" }
+ethereum-consensus = { git = "https://github.com/ralexstokes/ethereum-consensus", rev = "12508c1f9b0c8f4bf4c5e9b6d441e840c1b37fd9" }
```

Comment:
I had to use full commit hash for this particular version for some reasons, it doesn't compile if I use the short commit hash.

Reply:
This commit fails (using short commit hash):
https://github.com/ralexstokes/mev-rs/commit/98966663148046b1315cfa24e8bf453dd67b08b9

But this one (using full commit hash) passes:
https://github.com/ralexstokes/mev-rs/commit/dedc77a796986603fb3376c5f353863d09e0dbf2

Reply:
strange, but thanks